rust/src/lib.rs

```diff
@@ -30,10 +31,10 @@ pub use verify::{CustomVerifier, Verifier, VerifierBuilder};
 ///     .sign()?;
 /// # Ok(()) }
 /// ```
-pub fn sign_with_pem<'a>(
-    kid: &'a str,
-    private_key_pem: &'a [u8],
-) -> SignerBuilder<'a, &'a str, &'a [u8], Unset, Unset, Unset> {
+pub fn sign_with_pem(
+    kid: Uuid,
```

Comment:
This is a breaking change to the library right?

Reply:
Do we specify the KID will always be a UUID in our signing requirements?
Is there a good reason to make this change to the library, especially given other language libraries treat it as an opaque string?